docs/css/src/issues-list.css

```diff
@@ -10,6 +10,10 @@
   border-left: 10px solid;
 }
 
+.issue-url {
+  word-break: break-word;
```

Comment:
that is an invalid property. did you mean `break-all`? :)

Reply:
upsi, yes. sorry. will fix it - thank you :)

Reply:
<3